warehouse/models/mart/transit_database/bridge_organizations_x_county_geography.sql

```diff
@@ -0,0 +1,29 @@
+{{ config(materialized='table') }}
+
+WITH orgs AS ( -- noqa
+    SELECT *
+    FROM {{ ref('int_transit_database__organizations_dim') }}
+),
+
+geography AS ( -- noqa
+    SELECT *
+    FROM {{ ref('int_transit_database__county_geography_dim') }}
+),
+
+bridge_organizations_x_county_geography AS (
+    SELECT
+        orgs.source_record_id AS source_record_id,
```

Comment:
I think we'd want `orgs.key AS organization_key` kind of following this: https://github.com/cal-itp/data-infra/blob/main/warehouse/models/mart/transit_database/bridge_organizations_x_ntd_agency_info.sql.